lib/tasks/anonymise_emails.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+desc "Set all email domain addresses to given address or default one." \
+      " Usage: `rake anonymise_emails TEST_DOMAIN=test@testmedefra.gov.uk`"
+task anonymise_emails: :environment do
```

Comment:
I would prefer to move this to the `lib/tasks/email.rake` task so that it comes under the `email` namespace eg `bundle exec rake email:anonymise`.

Do you have any issues with that @cintamani ?

Reply:
Nope!! Sounds good! :)